packages/perseus/src/perseus-types.ts

```diff
@@ -474,6 +474,56 @@ export type GraphRange = [
     y: [min: number, max: number],
 ];
 
+export type GrapherAnswerTypes =
+    | {
+          type: "absolute_value";
+          coords: [
+              // The vertex
+              Coord, // A point along one line of the absolute value "V" lines
```

Comment:
**nit**: I _think_ this comment is for the second `Coord` and should be dropped onto a newline.

Reply:
For sure. I just copied it exactly how it was from the previous location. I think I'll be going over the widgets again, so I can adjust it later if needed.